blocks/library/quote/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { html, query } = hpq;
+
+registerBlock( 'core/quote', {
+	title: wp.i18n.__( 'Quote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: ( node ) => query( 'blockquote > p', html() )( node ).join(),
+		citation: html( 'footer' )
+	},
+
+	edit( { attributes, setAttributes } ) {
+		const { value, citation } = attributes;
+
+		return (
+			<blockquote className="blocks-quote">
+				<Editable
+					value={ value }
+					onChange={ ( newValue ) => setAttributes( { value: newValue } ) } />
+				<footer>
+					<Editable
+						value={ citation }
+						onChange={ ( newValue ) => setAttributes( { citation: newValue } ) } />
+				</footer>
+			</blockquote>
+		);
+	},
+
+	save( attributes ) {
+		const { value, citation } = attributes;
+		return (
+			<blockquote>
```

Comment:
>🤔 How do we apply the `dangerouslySetInnerHTML` without a wrapper?

Yeah, we ought to be thinking about this, because I don't think it's particularly obvious and the "dangerous" terminology is quite off-putting.

Perhaps we can walk the tree returned from the function and manually revise string children as `dangerouslySetInnerHTML`? 😬